### ISSUE TITLE
Skip empty rows does not work

### ISSUE THREAD

**Issue by isalEfisco** [shuchkin/simplexlsx#56]
Hi, `SimpleXLSX::parse` with "skip_empty_rows = true" return arrays with empty values.

**Comment by shuchkin** [comment]
```
10|11| 12
20|  | 22
  |  |     
40|41|42
```
$xlsx->skip_empty_rows = true;
$xlsx->rows();

```
10|11|12
20|  |22
40|41|42
```

IF u get the other result
send your xlsx to sergey.shuchkin@gmail.com

**Comment by isalEfisco** [comment]
Hi, I sent

**Comment by shuchkin** [comment]
fixed in 0.8.10, check it
